bin/custom/concerttours/src/concerttours/daos/impl/DefaultBandDAO.java

```diff
@@ -0,0 +1,38 @@
+package concerttours.daos.impl;
+
+import concerttours.daos.BandDAO;
+import concerttours.model.BandModel;
+import de.hybris.platform.servicelayer.search.FlexibleSearchQuery;
+import de.hybris.platform.servicelayer.search.FlexibleSearchService;
+
+import java.util.List;
+
+public class DefaultBandDAO implements BandDAO {
+
+    private FlexibleSearchService flexibleSearchService;
+
+    public void setFlexibleSearchService(final FlexibleSearchService flexibleSearchService)
+    {
+        this.flexibleSearchService = flexibleSearchService;
+    }
+
+    @Override
+    public List<BandModel> findBands() {
+        final String queryString = //
```

Comment:
Always extract string literals to constants (the exception is pretty much only logging/throwable messages). Also what's about this //, why?